WORKSPACE

```diff
@@ -237,3 +237,14 @@ load("@io_bazel_rules_dotnet//dotnet:defs.bzl", "dotnet_register_toolchains", "d
 dotnet_register_toolchains()
 
 dotnet_repositories_nugets()
+
+git_repository(
```

Comment:
Use `http_archive` with `https://github.com/bazelbuild/rules_perl/archive/refs/tags/0.1.0.zip` (and also mirror to our GCS bucket).

In general, avoid using `git_repository` whenever possible. When necessary, prefer to use `commit` instead of `branch` to avoid upstream surprises.

Reply:
I can't: https://github.com/EngFlow/example/pull/57#issuecomment-1339502506

Reply:
Ah, sorry, missed that. It looks like they've fixed that in `022b8daf2bb4836ac7a50e4a1d8ea056a3e1e403`.

In that case, let's use `git_repository` with `commit`.